server/plugins/renderer/renderer.module.js

```diff
@@ -12,12 +12,15 @@ const templateAssembler = require('../../../lib/template-assembler');
 const utils = require('../../lib/utils');
 const { readFromStream } = require('../../../lib/utils/asyncUtils');
 const NetworkUtils = require('../../../lib/utils/NetworkUtils');
+const contentApiClient = require('../../../lib/content-api-client');
+const { getPageType, isValidPageTypeEntityIdReq } = require('../../lib/page-type-util');
 
 const networkUtils = new NetworkUtils();
 
 const internals = {
     options: {},
     cacheTTL: 1000 * 15, // 15 seconds
+    graphQLCacheTTL: 1000 * 300, // 5 minutes
```

Comment:
Just a thought for the future, it might be worth considering literally never refreshing the cache without user intervention, or making it a cli setting or something. Just imagining in the vast majority of use cases the user wouldn't be expecting or needing the widgets to update. (I don't really have a sense for how many users are using stencil cli at any given time, admittedly it's probably only a handful so not a huge deal at the moment)

Reply:
There is an option to start the stencil-cli without using cache `stencil start --no-cache`. So it might be work changing the TTL to say 60mins or more. I think the TTL is not optional and would throw an error if not provided.